[AY1920S1-CS2113T-F11-1] SpongeBob

Tejasvas123 opened the pull request:
@Zhubo225 
@KevinAlvarez7 
@swang1997

j-lum committed:
Add support for Gradle workflow

j-lum committed:
Add sample checkstyle configuration

Add toolVersion block in to Gradle code sample to prevent errors.

j-lum committed:
Change file mode on `gradle` to be executable

j-lum committed:
Merge pull request #12 from j-lum/gradle+x

Change file mode on `gradle` to be executable (#9)

j-lum committed:
Add configuration for console applications

Gradle defaults to an empty stdin which results in runtime exceptions
when attempting to read from `System.in`. Let's add some sensible
defaults for students who may still need to work with the standard
input stream.

j-lum committed:
Merge pull request #13 from j-lum/javaexec

Add configuration for console applications

Tejasvas123 committed:
Merge branch 'gradle'
Required for automating build process

Tejasvas123 committed:
added level 1 with indentation

Tejasvas123 committed:
added level 2

Tejasvas123 committed:
added Task.java

Tejasvas123 committed:
Level-3

Tejasvas123 committed:
Added level-4 using inheritance

Tejasvas123 committed:
Handled some errors related to Duke

Tejasvas123 committed:
Trimmed code and completed Level-7

Tejasvas123 committed:
added date time specifications to Duke

Tejasvas123 committed:
Add Delete Task feature

Tejasvas123 committed:
Test some basic input output cases for Duke

Tejasvas123 committed:
Add delete feature

Tejasvas123 committed:
Find tasks that match a keyword

Tejasvas123 committed:
Resolve conflicts between level-6 and level-9

Tejasvas123 committed:
Improve OOP design and refactor code

Tejasvas123 committed:
Fix typo

Tejasvas123 committed:
Improve style

Tejasvas123 committed:
Handle some exceptions

Tejasvas123 committed:
Add java doc and handle checkstyle errors

Tejasvas123 committed:
Add some JUnit Testing

Tejasvas123 committed:
Handle checkstyle errors

Tejasvas123 committed:
Divide duke into various packages

Tejasvas123 committed:
Add some test cases to check for date specifications

Tejasvas123 committed:
Add packages to JUnit testing

Tejasvas123 committed:
bugFix old version

Tejasvas123 committed:
Small bugFix

Tejasvas123 committed:
Fix checkstyle errors

Tejasvas123 committed:
Merge branch 'master' of https://github.com/Tejasvas123/main

Tejasvas123 committed:
configure testing

Tejasvas123 committed:
add travis configuration

Tejasvas123 committed:
configure gradle

Tejasvas123 committed:
Add steps for contributing to SpongeBob

Tejasvas123 committed:
Add a note to contributing.md (#1)

Tejasvas123 committed:
configure gradle (#3)

Tejasvas123 committed:
Improve coding style (#4)

Tejasvas123 committed:
[CS2113T-F11-1]-Tejasvas123-A-Jar (#5)

* Improve coding style

* Add Do-After Task

* Handle checkStyle errors

* make a release

Tejasvas123 committed:
Added some test cases for DoAfter Command (#6)

* Improve coding style

* Add Do-After Task

* Handle checkStyle errors

* make a release

* Add test cases for DoAfter

Tejasvas123 committed:
[CS2113T-F11-1]-Tejasvas123-B-FixedDurationTasks (#7)

* Improve coding style

* Add Do-After Task

* Handle checkStyle errors

* make a release

* Add test cases for DoAfter

* Enable SpongeBob to recognize FixDurationTask

* Minor BugFix

Tejasvas123 committed:
[CS2113T-F11-1]-Tejasvas123-B-DoAfterTasks (#8)

* Improve coding style

* Add Do-After Task

* Handle checkStyle errors

* make a release

* Add test cases for DoAfter

* Enable SpongeBob to recognize FixDurationTask

* Minor BugFix

* Enable testing for FixedCommand

Tejasvas123 committed:
[CS2113T-F11-1]-Tejasvas123-B-RecurringTasks (#10)


* Handle checkstyle and renamed files

Tejasvas123 committed:
[CS2113T-F11-1]-Tejasvas123-A-Travis (#11)

* Improve coding style

* Add Do-After Task

* Handle checkStyle errors

* make a release

* Add test cases for DoAfter

* Enable SpongeBob to recognize FixDurationTask

* Minor BugFix

* Enable testing for FixedCommand

Zhubo225 committed:
Enable Duke to snooze/postpone/reschedule task

swang1997 committed:
Merge pull request #13 from Zhubo225/SnoozeCommand

[CS2113T-F11-1]-Zhubo225-B-Snooze

swang1997 committed:
[CS2113T-F11-1]-swang1997-B-DoWithinPeriodTasks (#15)

* DoWithin adds a period task

* DoWithin adds a period task

* DoWithin adds a period task

KevinAlvarez7 committed:
[CS2113T-F11-1]-KevinAlvarez7-B-Reminders (#16)

* Reminders' Tasks

* Reminders Tasksv1.1

* Reminders Tasksv1.2 checkstyle test completed

* J-Unit Testing

* J-Unit Testing

* J-Unit Testing

Tejasvas123 committed:
Delete viewSchedule (#17)

* Improve coding style

* Add Do-After Task

* Handle checkStyle errors

* make a release

* Add test cases for DoAfter

* Enable SpongeBob to recognize FixDurationTask

* Minor BugFix

* Enable testing for FixedCommand

* Delete viewschedule.java

swang1997 committed:
Added Admin Docs

swang1997 committed:
Merge pull request #20 from swang1997/master

Added Admin Docs

swang1997 committed:
Added Test Image

swang1997 committed:
Update README.adoc

swang1997 committed:
Update README.adoc

swang1997 committed:
Added test image and image folder

swang1997 committed:
Edited Frame of User_Guide

swang1997 committed:
Merge pull request #22 from swang1997/master

Added Frame of Userguide and Image Folder

Zhubo225 committed:
Update RemindCommandTest and SnoozeCommandTest (#19)

Tejasvas123 committed:
Modify gitignore (#23)

* Improve coding style

* Add Do-After Task

* Handle checkStyle errors

* make a release

* Add test cases for DoAfter

* Enable SpongeBob to recognize FixDurationTask

* Minor BugFix

* Enable testing for FixedCommand

* add text files to gitignore

swang1997 committed:
Updated commands in User Guide

swang1997 committed:
Merge branch 'master' into master

swang1997 committed:
Merge pull request #24 from swang1997/master

Updated commands in User Guide

Zhubo225 committed:
Developer Guide v1.0

Tejasvas123 committed:
Merge branch 'master' into DG

Zhubo225 committed:
Merge pull request #27 from Zhubo225/DG

[AY1920S1-CS2113T-F11-1] Developer Guide

Tejasvas123 committed:
Update README.adoc

Tejasvas123 committed:
Update README.adoc

Modified Readme to make it look like it is a legit fully-developed product

Zhubo225 committed:
Update ABOUT_US.adoc (#28)

swang1997 committed:
added ui image (#29)

* added ui image

* deleted Q&A

Tejasvas123 committed:
Update README.adoc

Tejasvas123 committed:
Update README.adoc

Tejasvas123 committed:
Update USER_GUIDE.adoc

swang1997 committed:
added developer images (#30)

* added ui image

* deleted Q&A

* added image for developers

* Create README.adoc

swang1997 committed:
changed image format (#31)

* added ui image

* deleted Q&A

* added image for developers

* Create README.adoc

* Changed image format

* changed image format

Tejasvas123 committed:
Update USER_GUIDE.adoc

Zhubo225 committed:
Update AboutUs (#32)

Tejasvas123 committed:
Update USER_GUIDE.adoc

Tejasvas123 committed:
Update USER_GUIDE.adoc

Tejasvas123 committed:
Update USER_GUIDE.adoc

Zhubo225 committed:
Updated Developer Guide (#69)

Tejasvas123 committed:
Rename ABOUT_US.adoc to AboutUs.adoc

swang1997 committed:
Moved README into main folder (#71)

* added ui image

* deleted Q&A

* added image for developers

* Create README.adoc

* Changed image format

* changed image format

* Update README.adoc

moved README into main folder

swang1997 committed:
Changed format of datetime (#76)

* added ui image

* deleted Q&A

* added image for developers

* Create README.adoc

* Changed image format

* changed image format

* Update README.adoc

moved README into main folder

* DateTime Format Changed

Changed from simple date format to LocalDateTime

* Update Parser.java

Tejasvas123 committed:
Configure json for storing data on hard disk (#77)

* Improve coding style

* Add Do-After Task

* Handle checkStyle errors

* make a release

* Add test cases for DoAfter

* Enable SpongeBob to recognize FixDurationTask

* Minor BugFix

* Enable testing for FixedCommand

* add text files to gitignore

* Configure gradle for javafx

* ignore text and data files related to json

* BugFix DateDowithin and typo in errors

* store list of tasks as a List

* configure JSON for Filehandling

Tejasvas123 committed:
Update README.adoc (#79)

swang1997 committed:
Changed UI (#80)

* added ui image

* deleted Q&A

* added image for developers

* Create README.adoc

* Changed image format

* changed image format

* Update README.adoc

moved README into main folder

* DateTime Format Changed

Changed from simple date format to LocalDateTime

* Update Parser.java

* Update Ui.png

* make changes

Tejasvas123 committed:
Update README.adoc (#81)

Tejasvas123 committed:
Update USER_GUIDE.adoc (#82)

Tejasvas123 committed:
Morphing to sponge bob (#84)

* Improve coding style

* Add Do-After Task

* Handle checkStyle errors

* make a release

* Add test cases for DoAfter

* Enable SpongeBob to recognize FixDurationTask

* Minor BugFix

* Enable testing for FixedCommand

* add text files to gitignore

* Configure gradle for javafx

* ignore text and data files related to json

* BugFix DateDowithin and typo in errors

* store list of tasks as a List

* configure JSON for Filehandling

* put the files related to duke in a temp folder

* Rebase code

* add basic add/delete for spongeBob

* enable spongebob to add delete and edit lockers

Zhubo225 committed:
DG2.0 (#85)

* UIstuff

* DeveloperGuide2.0

* DG2.0

* modify dg

Tejasvas123 committed:
Refactor parser #1 (#87)

* make parser command non-static

* Initial Parser structure for SpongeBob

* fix checkstyle

andyrobert3 commented on src/main/java/duke/Duke.java:
```diff
@@ -0,0 +1,59 @@
+    private Parser parser;
+
+    /**
+     * This constructor instantiates the Duke class by loading data from a file.
```
This header comment does not follow convention (i.e. first sentence should be something like `Selects next line …` instead of `Select next line` or `This method is for selecting …`)

andyrobert3 commented on src/main/java/duke/Duke.java:
```diff
@@ -0,0 +1,59 @@
+        }
+    }
+
+    /**
```
Similar to above's comment, header comment does not follow convention (i.e. first sentence should be something like Selects next line … instead of Select next line or This method is for selecting …)

andyrobert3 commented on src/main/java/duke/logic/commands/Command.java:
```diff
@@ -0,0 +1,17 @@
+import duke.ui.Ui;
+
+public abstract class Command {
+    public boolean isExit = false;
```
Does isExist need to be public variable given it has a getter method already?

andyrobert3 commented on src/main/java/duke/logic/commands/DeleteLockerCommand.java:
```diff
@@ -0,0 +1,32 @@
+    @Override
+    public void execute(LockerList lockerList, Ui ui, FileHandling storage) throws DukeException {
+        try {
+            int temp = Integer.parseInt(splitInput.get(0)) - 1;
```
variable naming for temp could be more descriptive

andyrobert3 commented on src/main/java/duke/logic/commands/DeleteLockerCommand.java:
```diff
@@ -0,0 +1,32 @@
+            ui.deleteMessage(lockerList.getAllLockers(), lockerA);
+            storage.saveData(lockerList);
+        } catch (NumberFormatException obj) {
+            throw new DukeException(" OOPS! Enter a positive integer after \"delete\"");
```
May want to integrate all the user messages into its own class to prevent magic strings

andyrobert3 commented on src/main/java/duke/models/LockerList.java:
```diff
@@ -0,0 +1,52 @@
+
+    @JsonGetter("lockers")
+    public List<Locker> getLockerList() {
+        return lockerList;
```
Returning List will pass by reference causing the object that calls 

`getLockerList()` to tamper with the `lockerList`, may want to reconsider to create a copy of `lockerList`

andyrobert3 commented on src/main/java/duke/models/Tag.java:
```diff
@@ -0,0 +1,47 @@
+        this.tagName = tagName;
+    }
+
+    public boolean checkValidTagName(String test) {
```
input `test` name could be more descriptive

andyrobert3 commented on src/main/java/duke/parser/AddBatchCommandParser.java:
```diff
@@ -0,0 +1,43 @@
+     * @throws DukeException when the command syntax is invalid
+     */
+    public Command parse(String fullCommand) throws DukeException {
+        if (fullCommand.trim().length() == 8) {
```
Magic Number of `8`, may want to store it as a `public static final` variable

andyrobert3 commented on src/main/java/duke/parser/Parser.java:
```diff
@@ -0,0 +1,41 @@
+            return new EditLockerCommandParser().parse(fullCommand);
+        } else  {
+            throw new DukeException(" Invalid command");
+        }
```
For readability, may want to use `switch` statement instead of `if else`

andyrobert3 commented on src/main/java/duke/ui/Ui.java:
```diff
@@ -0,0 +1,195 @@
+     * This function prints the hello message every time Duke is initiated.
+     */
+    public void showWelcome() {
+        String logo = "SpongeBob";
```
Why do you need to save `SpongeBob` as a variable if it is only used once inside the function

andyrobert3 commented on temp/duketest/command/ByeCommandTest.java:
```diff
@@ -0,0 +1,23 @@
+    @Test
+    public void checkBye() throws DukeException {
+        Command c = new ByeCommand();
+        assertFalse(c.isExit);
```
may want to use isExit() method instead of accessing member variable directly

andyrobert3 commented on temp/temp/command/AddDeadlineCommand.java:
```diff
@@ -0,0 +1,47 @@
+                    + "to separate task and time");
+        } else if (split2.trim().length() == 0) {
+            throw new DukeException(" Please enter the time frame");
+        }
```
Possible to increase SLAP here?
Also try to give more descriptive names for variables instead of `i` and `k`

andyrobert3 commented on temp/temp/command/AddDoAfterCommand.java:
```diff
@@ -0,0 +1,47 @@
+                    + " task and time");
+        } else if (split2.trim().length() == 0) {
+            throw new DukeException(" Please enter the time frame");
+        }
```
Possible to increase SLAP here?

andyrobert3 commented on temp/temp/command/DeleteCommand.java:
```diff
@@ -0,0 +1,31 @@
+    @Override
+    public void execute(TaskList tasks, Ui ui, FileHandling storage) throws DukeException {
+        try {
+            int temp = Integer.parseInt(splitInput.get(1)) - 1;
```
May want to provide a more descriptive name than temp

andyrobert3 commented on temp/temp/command/SnoozeCommand.java:
```diff
@@ -0,0 +1,84 @@
+        this.inputTask = inputTask;
+    }
+
+    private void updateSnooze(TaskList tasks, Ui ui, FileHandling storage,Task task,
```
Try to add comments for non-trivial methods

andyrobert3 commented on temp/temp/command/SnoozeCommand.java:
```diff
@@ -0,0 +1,84 @@
+                    + " task and snoozed/postponed/rescheduled time");
+        } else if (split1.trim().length() == 0) {
+            throw new DukeException(" Please enter the new time frame");
+        }
```
DRY principle can be followed to minimize duplicate code

andyrobert3 commented on temp/temp/parser/Parser.java:
```diff
@@ -0,0 +1,68 @@
+        } else if (inputTask.equals("find")) {
+            return new FindCommand(splitInput);
+        } else {
+            throw new DukeException(" Please enter a valid command");
```
`switch case` may improve readability compared to `if else`

andyrobert3 reviewed (commented):
Generally good progress on development

May want to take into account three things:
1. SLAP 
2. DRY principle - reduce copying & pasting code
3. Header comments - try to follow convention

Please Close PR after reading comments

Tejasvas123 committed:
Refactor parser #2 (#88)

* Improve coding style

* Add Do-After Task

* Handle checkStyle errors

* make a release

* Add test cases for DoAfter

* Enable SpongeBob to recognize FixDurationTask

* Minor BugFix

* Enable testing for FixedCommand

* add text files to gitignore

* Configure gradle for javafx

* ignore text and data files related to json

* BugFix DateDowithin and typo in errors

* store list of tasks as a List

* configure JSON for Filehandling

* make parser command non-static

* Initial Parser structure for SpongeBob

* fix checkstyle

* Refactor Parser to make it more OOP and support for Regex in order to have friendlier syntax

* BugFix checkStyle

Tejasvas123 committed:
Refactor parser #3 (#89)

* BugFix checkStyle

* finish refactoring parser

* delete unncesessary files

Tejasvas123 committed:
minor bugfix related to renaming and refactoring (#90)

* minor bugfix while renaming and refactoring

swang1997 committed:
Update DG (#126)

Tejasvas123 committed:
add jar file (#127)

Zhubo225 committed:
DG2.0 (#128)

KevinAlvarez7 committed:
Dev Guide Changes (#129)

Zhubo225 committed:
Dg.251019.2 (#132)

* Updated DG

* Updated DG

* Updated DG

* Updated DG

* Updated DG2.0

* Updated DG2.0

Tejasvas123 committed:
Add Ability for auto allocation of lockers (#135)

* refactor code to make it more OOP based and account for auto-allocation

* modify models to store student related details

* minor bugfix

* Modify parser in order to allow auto assigning

* override equals and hashCode() for models in order to account for checks regarding equality

* Add ability to autoallocate lockers

* minor bugFix in regex for email

* add ability to notify user when no free lockers were found in the listed preferences

* fix checkstyle errors

swang1997 committed:
Edited CSV Writer (#134)

* Merge branch 'master' of https://github.com/AY1920S1-CS2113T-F11-1/main

* Included CSV Writer

* Edited CSV Writer

* Amended changes to CSV Writer

* Update OpenCsv.java

Tejasvas123 committed:
Add ability to edit fields for locker (#136)

* refactor code to make it more OOP based and account for auto-allocation

* modify models to store student related details

* minor bugfix

* Modify parser in order to allow auto assigning

* override equals and hashCode() for models in order to account for checks regarding equality

* Add ability to autoallocate lockers

* minor bugFix in regex for email

* add ability to notify user when no free lockers were found in the listed preferences

* fix checkstyle errors

* Change some commands in order to improve architecture

* add ability to edit states of lockers

* modify edit locker

* add ability to edit usage fields

* minor bugfix for editusage and editlocker

* fix checkstyle errors

Tejasvas123 committed:
minor changes for CLI UI (#138)

* refactor code to make it more OOP based and account for auto-allocation

* modify models to store student related details

* minor bugfix

* Modify parser in order to allow auto assigning

* override equals and hashCode() for models in order to account for checks regarding equality

* Add ability to autoallocate lockers

* minor bugFix in regex for email

* add ability to notify user when no free lockers were found in the listed preferences

* fix checkstyle errors

* Change some commands in order to improve architecture

* add ability to edit states of lockers

* modify edit locker

* add ability to edit usage fields

* minor bugfix for editusage and editlocker

* fix checkstyle errors

* modify ui

* fix checkstyle

Zhubo225 committed:
Help Command Updated (#137)

* help command updated

* fix checkstyle

Tejasvas123 committed:
Configure Json (#140)

* configure json

* configure json and fix bugs related to it

* fix checkstyle errors

* minor bugFix

Tejasvas123 committed:
version 1.3

Tejasvas123 committed:
resolve merge conflicts

Tejasvas123 committed:
version 1.3 (#143)

KevinAlvarez7 committed:
Week 11 (#144)

* Dev Guide Changes

* week 11

Tejasvas123 committed:
modify user guide

Tejasvas123 committed:
resolve merge conflicts

Zhubo225 committed:
update ListCommand (#148)

Tejasvas123 committed:
Update ug (#146)

* version 1.3

* modify user guide

Tejasvas123 committed:
Update USER_GUIDE.adoc

Zhubo225 committed:
update Ui.png (#149)

Tejasvas123 committed:
add ability to delete usage and delete lockers

Tejasvas123 committed:
resolve merge conflicts

Tejasvas123 committed:
change ui

Tejasvas123 committed:
refactor usage to improve code quality and fix exception bugs

Tejasvas123 committed:
resolve date time errors

Tejasvas123 committed:
improve code quality by following OOP principles

Tejasvas123 committed:
restructure usage command

Tejasvas123 committed:
add support for storing and retrieving data from JSON file

Tejasvas123 committed:
fix checkstyle errors

KevinAlvarez7 committed:
Merge pull request #176 from Tejasvas123/update-locker

Update locker

Tejasvas123 committed:
Made file parsing more comprehensive (#178)

* change error message for zone (bugfix)

* make file parsing more comprehensive

Tejasvas123 committed:
J unit testing and Refactoring Storage (#182)

* change error message for zone (bugfix)

* make file parsing more comprehensive

* add basic j-unit testing for models

* refactor storage in order to make undo-redo easy

swang1997 committed:
Stats function included (#184)

* Stats function included

* Update Stats

* Update Stats.java

swang1997 committed:
Changed StatsCommand (#185)

* Stats function included

* Update Stats

* Update Stats.java

* Merge branch 'master' of https://github.com/AY1920S1-CS2113T-F11-1/main

* Changed StatsCommand

* Included student info while exporting

* Changed csv location

Tejasvas123 committed:
Cleaning code (#186)

* code cleanup

* fix checkstyle errors

Tejasvas123 committed:
Java doc (#189)

* modify javadoc for parser and storage

* modify javadoc for models

* add javadoc for logic.commands

* add sample data

* checkstyle

swang1997 committed:
Included Export CSV for selection of Tags (#187)

* Stats function included

* Update Stats

* Update Stats.java

* Merge branch 'master' of https://github.com/AY1920S1-CS2113T-F11-1/main

* Changed StatsCommand

* Included student info while exporting

* Changed csv location

* Included Selection of Tag for Exporting CSV

* merge conflict

* pass checkstyle

* clean code

* Fixed Minor Issues of Export Selection of CSV, adjusted export selection of parser files.

Fixed Minor Issues of Export Selection of CSV, adjusted export selection of parser files.

* added in case check

* included start and end date for both methods of exporting csv file.

* fixed checkstyle

* minor fixes to parser export

* Modified input for Export CSV selection

* Minor Adjustments

* Minor changes to code

KevinAlvarez7 committed:
FindSortRemind (#188)

* Dev Guide Changes

* week 11

* test v1.5

* Final-features:RemindersFindSort

* FindSortReminders

* fixed errors from last pr

* changes to reminders command regarding exceptions

Tejasvas123 committed:
Clear command (#190)

* add clear all command

* add java-doc

* resolve checkstyle errors

KevinAlvarez7 committed:
Updates to Sort and UG (#191)

* Dev Guide Changes

* week 11

* test v1.5

* Final-features:RemindersFindSort

* FindSortReminders

* fixed errors from last pr

* changes to reminders command regarding exceptions

* changes to sort: integer sorting and exception for empty list. Updated UG

Zhubo225 committed:
Final update for SELECT/UNDO/REDO/HISTORY/HELP commands and Ui (#193)

* Final update for SELECT/UNDO/REDO/HISTORY/HELP commands and Ui

* Final update for SELECT/UNDO/REDO/HISTORY/HELP commands and Ui

* Final update for SELECT/UNDO/REDO/HISTORY/HELP commands and Ui

Zhubo225 committed:
Updates UG (#195)

Tejasvas123 committed:
Logging (#196)

* minor bug fixes in code

* checkstyle

* minor changes

* gitignore log file

* minor changes

swang1997 committed:
Manual Testing in UG (#194)

* Stats function included

* Update Stats

* Update Stats.java

* Merge branch 'master' of https://github.com/AY1920S1-CS2113T-F11-1/main

* Changed StatsCommand

* Included student info while exporting

* Changed csv location

* Included Selection of Tag for Exporting CSV

* merge conflict

* pass checkstyle

* clean code

* Fixed Minor Issues of Export Selection of CSV, adjusted export selection of parser files.

Fixed Minor Issues of Export Selection of CSV, adjusted export selection of parser files.

* added in case check

* included start and end date for both methods of exporting csv file.

* fixed checkstyle

* minor fixes to parser export

* Modified input for Export CSV selection

* Minor Adjustments

* Minor changes to code

* Updated UG

* Edited UG DG

Added in Appendix E: Instructions for Manual Testing

* Added in Export and Stats Syntax for UI

* edited ug

Tejasvas123 committed:
Update User Guide (#197)

Tejasvas123 committed:
Update USER_GUIDE.adoc

Tejasvas123 committed:
Update USER_GUIDE.adoc

Tejasvas123 committed:
Update USER_GUIDE.adoc

Tejasvas123 committed:
Update USER_GUIDE.adoc

Tejasvas123 committed:
Update USER_GUIDE.adoc

Tejasvas123 committed:
minor-changes (#199)
